sessiontxn/isolation/readcommitted.go

```diff
@@ -251,7 +309,22 @@ func (p *PessimisticRCTxnContextProvider) AdviseOptimizeWithPlan(val interface{}
 		plan = execute.Plan
 	}
 
+	useLastOracleTS := false
 	if v, ok := plan.(*plannercore.Insert); ok && v.SelectPlan == nil {
+		useLastOracleTS = true
```

Comment:
It is possible for such inserts to read stale data/lock? e.g. in `insert ignore` or `insert on duplicate update`?

Reply:
I think, I should disable these two cases